arraycontext/impl/pyopencl/fake_numpy.py

```diff
@@ -175,6 +186,21 @@ def _flatten_array(ary):
                 f"with strides {ary.strides} of {ary.dtype}")
 
 
+def _cl_scalar_list_norm(ary, ord):
+    if ord is None:
+        ord = 2
+
+    from numbers import Number
+    if ord == np.inf:
+        return max(ary)
+    elif ord == -np.inf:
+        return min(ary)
```

Comment:
Huh, TIL. (OK to resolve when seen.)

Reply:
numpy seems to support this, so why not! I doesn't correspond to any actual normed space (to my knowledge), but it may be useful to bound things.